tests/providers/tpm.rs

```diff
@@ -32,7 +32,7 @@ lazy_static! {
                 .unwrap()
         }
     };
-    static ref MESSAGE: Vec<u8> = { b"Knights who say 'NI!'".to_owned().to_vec() };
+    static ref MESSAGE: Vec<u8> = b"Knights who say 'NI!'".to_owned().to_vec();
```

Comment:
🤣  did something change in Rust 1.44?

Reply:
Not sure, I updated my `rustc` and did not see the error 🤡